### DISCUSSION TITLE
Visual Studio login almost working?

### DISCUSSION THREAD

**Discussion by gonzaloamadio** [community/community#13726] [Copilot Conversations]
I installed the extension, Got a popup with the message "login to github". I clicked on it and login to github in chrome.

After that I got a token and the instructions:

"Copy the token.
Switch back to VS code.
Click Signing in to github.com... in the status bar.
Paste the token and hit enter."

But do not have the Signing in to Github, I have already clicked on it. And now I can not make it appear or I do not know where to put the token.

**Comment by davecheney** [comment]
Are you using VSCodium or OSS Code?

**Reply by gonzaloamadio** [reply]
Normal visual studio code in Mac. Like this one: https://code.visualstudio.com/docs?dv=osx

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by gonzaloamadio** [comment]
Did it. You have to check the "always allow ..  " in the pop up.

Anyway it is not working, but I think some security measure from my laptop (is pre-configured by my work) is blocking it. =(
